tools/check-templates

```diff
@@ -206,7 +206,9 @@ def check_html_templates(templates, modified_only):
     # Our files with .html extensions are usually for Django, but we also
     # have some for Casper tests and a few static ones.
     # The file base.html has a bit of funny HTML that we can't parse here yet.
-    templates = filter(lambda fn: 'base.html' not in fn, templates)
+    templates = filter(
+        lambda fn: ('base.html' not in fn) and ('casperjs' not in fn),
+        templates)
```

Comment:
Maybe update the comment about how the capserjs tests intentionall hve some that fail this?